src/components/Navigation.js

```diff
@@ -0,0 +1,56 @@
+import { NavLink } from "react-router-dom";
+
+import styled from "styled-components";
+
+function Navigation() {
+  return (
+    <NavUl>
+      <ul>
+        <li>
+          <StyledNavlink to="/warenkorb">Warenkorb</StyledNavlink>
+        </li>
```

Comment:
Are these routes accessible? I don't see a component for `warenkorb`

Reply:
Routes -yes, but Pages not. Its my next step to create another  Pages

Reply:
Thanks for review!

Reply:
Changes passed. goes to quality ...
https://github.com/nazardiachok/Capstone-New-Project/tree/navbar